source/data_operations.py

```diff
@@ -1,29 +1,34 @@
-import pandas as pd
+import keras as ke
+import keras.backend as K
+from keras.layers import Input, Dense, Dropout
+from keras.models import Model
 import numpy as np
-import matplotlib.pyplot as plt
-import seaborn as sns
-from sklearn.model_selection import train_test_split
-from sklearn.preprocessing import StandardScaler
-from sklearn.ensemble import RandomForestRegressor
-from sklearn.svm import SVC
-from sklearn.metrics import accuracy_score, classification_report, confusion_matrix, f1_score
+import pandas as pd
+
+def nn_classifier(n_features):
```

Comment:
It might be nice to have some of the parameters of the NN architecture either be passable as command line arguments or read from a YAML file.